projects/ad9371/src/app/headless.c

```diff
@@ -82,6 +82,10 @@
 extern ad9528Device_t clockAD9528_;
 extern mykonosDevice_t mykDevice;
 
+#ifdef DAC_DMA_EXAMPLE
+uint32_t dac_buffer[DAC_BUFFER_SIZE] __attribute__ ((aligned));
+uint16_t adc_buffer[ADC_BUFFER_SIZE*4] __attribute__ ((aligned));
```

Comment:
- *4 because of 4 channels (rx1i, rx1q, rx2i, rx2q)
- DAC_MAX_SAMPLES wouldn't make sense but maybe inline 4 could be replaced by a number of channels define or something like that @ranechita 
- different types because DAC functions expect uint32_t whereas ADC precision is uint16_t

Reply:
also i just noticed, maybe DAC_BUFFER_SIZE is not good naming, it should probably DAC_BUFFER_SAMPLES, same for ADC

Reply:
i will change those and add the define for the number of channels